content/en/monitors/types/process.md

```diff
@@ -19,22 +19,45 @@ further_reading:
 
 ## Overview
 
-Live process monitors are based on data collected by the [Process Agent][1]. Create monitors that warn or alert based on the count of any group of processes across hosts or tags.
+Live Process Monitors are based on data collected by the [Process Agent][1]. Create monitors that warn or alert based on the count of any group of processes across hosts or tags.
+
+Live Process Monitors are recommended for the following use cases:
+
+- Ensure that enough instances of a non-containerized long-lived process are running.
+- Flag when a specific process is executed.
+
+Note that only long lived processes are picked up by the Agent. This means that monitors on processes that live for less than 20 seconds may be flaky.
 
 ## Monitor creation
 
-To create a [live process monitor][2] in Datadog, use the main navigation: *Monitors --> New Monitor --> Live Process*.
+To create a [Live Process Monitor][2] in Datadog, use the main navigation: *Monitors --> New Monitor --> Live Process*.
 
 ### Select processes
 
-Search for a process to monitor using a space separated list of strings. The search performs a partial match, fuzzy search across all processes on your infrastructure. Search operators `AND`, `OR`, and `NOT` are supported. See the [Live Process Monitoring][3] page for details. Additionally, filter by tags to refine your monitor to a specific scope.
-
-Matching processes and counts are displayed below the search:
+You can use either tags or a fuzzy search to filter across all processes in your infrastructure. Matching processes and counts are displayed below the search:
 
 {{< img src="monitors/monitor_types/process/select_processes.png" alt="Select processes" style="width:90%;">}}
 
 After defining your search, a graph is displayed above the search inputs with an approximation of the total number of processes found. For more granular data, visit your [Live Process Page][4].
 
+#### Tags search
+
+Filter processes to monitor by their tags. Datadog recommends that you first try to filter processes by their tags before using full text search.
+
+#### Full text search
+
+If you cannot scope processes down to the granularity you would like using tags, it is possible to filter processes by their commandline. The full text search field performs a partial match, fuzzy search across all processes on your infrastructure. Search operators `AND`, `OR`, `NOT` are supported. See the [Live Process Monitoring][3] page for more details.
```

Comment:
FYI, text terms are matched against both command line and username

Reply:
Ah great point! I will add this to the documentation